### PR TITLE
Clicking on edit next to details on the conversation show screen just returns to the show screen.

### PR DESCRIPTION
**Pull request by smn**
We need it to go to a view for editing details.

### CODE REVIEW

**Comment by smn** [comment]
this one?

vumi_go

**Comment by hodgestar** [comment]
Yep.

**Comment by smn** [comment]
I'm guessing that having a template-level option of seeing whether a given conversation is editable would be better long term.

**Comment by smn** [comment]
My thinking is that that can possibly be addressed better when all views have view_definitions, otherwise we're layering too many hacks.

File: go/conversation/templates/conversation/edit_detail.html [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+{% endblock %}
+
+{% block extramedia %}
+    {{ edit_form_media }}
+{% endblock %}
+
+{% block ondomready %}
+    $('.content .actions .right button').click(function() {
+        $('#form-conversation').submit();
+    });
```
Comment (hodgestar) [inline go/conversation/templates/conversation/edit_detail.html:33]: Maybe it would be better for the button to have an id rather than being identified by it's position within the page hierarchy?

**Comment by hodgestar** [comment]
:+1: other than the suggestion about the id. Some tests would also be good but I'm not going to let that stop the branch landing at this point.

File: go/conversation/templates/conversation/edit_detail.html (hunk 1 above)
Comment (smn) [inline go/conversation/templates/conversation/edit_detail.html:33]: True but this comes straight from the existing [edit.html template](https://github.com/praekelt/vumi-go/blob/537c0b3e4af4903e189f079a16179f41c28ae18d/go/conversation/templates/conversation/edit_detail.html#L31-L33) and possibly it'd be better to address that in a separate PR? Not sure.

Comment (hodgestar) [inline go/conversation/templates/conversation/edit_detail.html:33]: Happy for it to be addressed separately or left for now. The person who adds a cancel button can fix it.
